widget/button.go

```diff
@@ -179,7 +201,8 @@ func (b *Button) MouseOut() {
 func (b *Button) MouseMoved(*desktop.MouseEvent) {
 }
 
-// CreateRenderer is a private method to Fyne which links this widget to its renderer
+// CreateRenderer is a private method to Fyne which returns a new renderer for this widget type.  To refresh an existing
+// widget, use the `widget.Renderer` function with a pointer to the existing widget as the argument.
```

Comment:
I think you mean widget.Refresh()

Reply:
I'm just going to revert these comments, we talked about adding them when I first started working on this (before the icon colorization feature) but at this point I'd rather not have them be part of this PR if that's cool. Fixed, pushed.